playwright-tests/create-fuels.test.ts

```diff
@@ -25,3 +25,28 @@ test('counter contract - increment function call works properly', async ({ page
   const counter = page.getByTestId('counter');
   await expect(counter).toBeVisible();
 });
+
+test('faucet page', async ({ page }) => {
+  await page.goto('http://127.0.0.1:3000/faucet', { waitUntil: 'networkidle' });
+
+  await page.waitForTimeout(2000);
+
+  // check if the two fields are pre-filled
+  const receiverAddressInput = page.getByLabel('Receiving address:');
+  await expect(receiverAddressInput).not.toBeEmpty();
+  const receiverAddress = await receiverAddressInput.inputValue();
+
+  const amountToSendInput = page.getByLabel('Amount (ETH):');
+  await expect(amountToSendInput).toHaveValue('5');
+
+  // click the send funds button
+  const sendFundsButton = page.getByText('Send Funds');
+  await sendFundsButton.click();
+
+  await page.waitForTimeout(500);
+
+  const successToast = page.getByText('Funds sent!');
+  await expect(successToast).toBeVisible();
+
+  // TODO: Validate and assert that the balance has been updated. We need to use `fuels` as a dependency to do this.
```

Comment:
Filed an issue for this todo: #2494

Reply:
@Dhaiwat10 Could you please further elaborate on the issue description? 
 - https://github.com/FuelLabs/fuels-ts/issues/2494

It isn't obvious why we can't use `fuels` as a dependency or why exactly that would be necessary. Also, if you have an insight into a possible hypothesis/solution, that would be helpful.

Reply:
@arboleya added some context there.

Reply:
@petertonysmith94 I am not sure if that would be possible, but might be worth a try. Would need a few changes in the playwright config.

Reply:
Resolving this conversation for now since I have filed an issue for this